src/views/waypoint/home/server.ts

```diff
@@ -0,0 +1,86 @@
+import slugify from 'slugify'
+// TODO: need to discuss from whence we should
+// TODO: source content down the road. For now,
+// TODO: sourcing from JSON for demo purposes.
+import TEMP_CONTENT from './content.json'
+
+const basePath = 'docs'
+const productName = 'Waypoint'
+const productSlug = 'waypoint'
+
+async function getStaticProps(): Promise<$TSFixMe> {
+  const CONTENT = TEMP_CONTENT as $TSFixMe
+
+  // TODO: Content blocks content should likely
+  // TODO: be fetched dynamically, currently things
+  // TODO: like the title and description of docs pages
+  // TODO: are hard-coded alongside the URL to link to.
+  // TODO: Should be possible to fetch the title and description
+  // TODO: at build time from said URL, to ensure the displayed
+  // TODO: content is accurate, while not needing to be
+  // TODO: manually kept up to date?
+  CONTENT.blocks = CONTENT.blocks.map((block) => {
+    const { type, heading } = block
+    if (type !== 'heading') return block
+    return {
+      ...block,
+      slug: slugify(heading, { lower: true }),
+    }
+  })
+
+  const navData = [
```

Comment:
Customized `navData` for the `/waypoint` product landing, again to match Figma (was previously the full `docs` nav data). Have added `TODO` comments above to signal that we need to refine content sourcing here.

Definitely expecting this to change (esp the `_demoExternalLink` part), this is very much targeted at getting things ready to demo.

Reply:
The code in my sidebar PR renders an external link icon when the `href` prop is present, so I think this should "just work" without the `_demoExternalLink` property once this is working off of those changes.